openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -27320,6 +27320,95 @@ OS:Generator:FuelSupply,
       \minimum 0.0
       \maximum 1.0
 
+OS:Generator:PVWatts,
+      \min-fields 4
+      \memo Describes a simple set of inputs for an array of photovoltaic (PV) modules as described
+      \memo in the PVWatts software. A series of different PVWatts arrays can be connected to a
+      \memo single electric load center (preferably through an
+      \memo ElectricLoadCenter:Inverter:PVWatts). Array tilt and azimuth can be either specified
+      \memo on this object or taken from a referenced building surface or shading surface. If
+      \memo a surface is specified, the array participates normally in all shading calculations.
+  A1, \field Handle
+       \type handle
+       \required-field
+  A2, \field Name
+      \required-field
+      \reference GeneratorNames
+  A3, \field PVWatts Version
+      \type choice
+      \key 5
+  N1, \field DC System Capacity
+      \note Nameplate rated DC system capacity in watts
+      \type real
+      \required-field
+      \units W
+      \minimum> 0
+  A4, \field Module Type
+      \type choice
+      \key Standard
+      \key Premium
+      \key ThinFilm
+      \default Standard
```

Comment:
Thank you for making a default, you are getting the hang of this :-)

Reply:
np